test/integration/support/protractor-config/jasmine-spec-reporter.conf.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
```

Comment:
I'd remove this file unless we need it since (we don't have an integration test that actually uses it)